lib/spack/spack/test/environment.py

```diff
@@ -135,23 +122,15 @@ def test_filter_system_paths(miscellaneous_paths):
     filtered = filter_system_paths(miscellaneous_paths)
     expected = [
         '/usr/local/Cellar/gcc/5.3.0/lib',
+        '/usr/local/lib',
+        '/usr/local/include',
+        '/usr/local/lib64',
```

Comment:
is it a github issue or why do you change the test of `filter_system_paths`? You don't change the function so the test should remain valid.